### PR TITLE
Create 2015-10-29-announcing-capablanca-release-candidate-1.html.md

### PR DESCRIPTION
**Pull request by bproffitt**
Blog announcement for 10/29 release of Capablanca RC1

### CODE REVIEW

File: source/blog/2015-10-29-announcing-capablanca-release-candidate-1.html.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Announcing Capablanca Release Candidate 1
+date: 2015-10-29 14:03 UTC
+author: chessbyte
+tags: release Capablanca
+comments: true
+published: true
+---
+We're pleased to announce that, effective today, the first [Capablanca release candidate](http://manageiq.org/download/devel/) is now available. Among the improvements in this release:
```
Comment (jrafanie) [inline source/blog/2015-10-29-announcing-capablanca-release-candidate-1.html.md]: @bproffitt 

Unless I'm misunderstanding or there's more website changes coming... but I think this is wrong.  This links to a page with links to to each of the image platforms, which for vmware, links to http://manageiq.org/download/manageiq-vsphere-devel.ova which downloads the latest nightly build off of master:

```
wget http://manageiq.org/download/manageiq-vsphere-devel.ova
...
Location: http://releases.manageiq.org/manageiq-vsphere-master-20151026-d5305dacb7.ova
...
Location: http://82396214b15770b3002a-d3fd4008af5431479c47090ec0cc5a4d.r13.cf5.rackcdn.com//manageiq-vsphere-master-20151026-d5305dacb7.ova
```

Perhaps we need to link to the manageiq-openstack-capablanca-1-rc1 images here:  http://releases.manageiq.org/?

**Comment by matthewd** [comment]
Are we just listing changes relative to the beta? If so, we should probably make that clear.. the whole point of an RC is that we're expecting people to try it, who have not been following the earlier preview releases.

In fact, if we are going to treat the RC as incremental, this announcement should probably have an explicit "see \[the alpha announcement\] for a summary of all changes since botvinnik".

**Comment by Fryguy** [comment]
I thought I made this clear in emails, but I'll reiterate.

The website needs to be changed to have 3 modes... nightly, stable, and rc.  I believe this mode is already in the code, just commented out.  

Nightly should point to the nightly, stable to botvinnik, and rc should point to the new capablanca-1-rc1.

Once we officially release capablanca, that becomes stable, and rc gets hidden again.

The blog post should point to the links on the site, *not* to releases.manageiq.org

**Comment by bproffitt** [comment]
You did make that clear, but that functionality is not available yet, and
it is my understanding that the website designer was not able to implement
it due to some questions he had for the ManageIQ team that had not been
answered.

Unfortunately that has been rendered moot; the designer is currently on
PTO. I have copied his manager on this thread if you want to pursue that
discussion.

In the meantime, I can certainly change the link to the website page. As
for the listed features, I was given a copy of the Sprint 31 slide
presentation to use as a headstart until the final change log was done. If
those features are incremental from beta and not the last GA, then clearly
I will have to wait until the best source document is available and list
the features from that.

BKP

On Wednesday, October 28, 2015, Jason Frey <notifications@github.com> wrote:

> I thought I made this clear in emails, but I'll reiterate.
>
> The website needs to be changed to have 3 modes... nightly, stable, and
> rc. I believe this mode is already in the code, just commented out.
>
> Nightly should point to the nightly, stable to botvinnik, and rc should
> point to the new capablanca-1-rc1.
>
> Once we officially release capablanca, that becomes stable, and rc gets
> hidden again.
>
> The blog post should point to the links on the site, *not* to
> releases.manageiq.org
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq.org/pull/246#issuecomment-151969385>
> .
>


-- 

Brian Proffitt
574-383-9BKP
Twitter: TheTechScribe
Skype: LinuxScribe
www.proffitt.org

**Comment by bproffitt** [comment]
Updated feature list. Let me know what download link you would like to use.

**Comment by chrispy1** [comment]
Hi Brian,  
Feedback from engineering is if we could use the words "since" and "beta".  Also adding a link to the Alpha announcement as a general list of what we've included since Botvinnik would be helpful.

Thanks,
Chris

**Comment by bproffitt** [comment]
That feedback may be moot since I updated the document to actually include
changes since Botvinnik, which was how I interpreted their request.

BKP


Brian Proffitt
574-383-9BKP
Twitter: TheTechScribe
Skype: LinuxScribe
www.proffitt.org

**Comment by Fryguy** [comment]
Once #247 is merged, you can link to http://manageiq.org/download/prerelease/

**Comment by Fryguy** [comment]
cc @jzb

**Comment by bproffitt** [comment]
Noted, thanks!

BKP


Brian Proffitt
574-383-9BKP
Twitter: TheTechScribe
Skype: LinuxScribe
www.proffitt.org

**Comment by Fryguy** [comment]
#247 is merged.

**Pull request closed without merging**